src/storage/src/source/persist_source.rs

```diff
@@ -162,13 +162,14 @@ where
     let (inner, token) = crate::source::util::source(
         scope,
         format!("persist_source {:?}: batch distribution", source_id),
+        None,
         move |info| {
             let waker_activator = Arc::new(scope.sync_activator_for(&info.address[..]));
             let waker = futures::task::waker(waker_activator);
 
             let mut current_ts = 0;
 
-            move |cap_set, output| {
+            move |cap_set, output, _| {
```

Comment:
nit: could use sth like `_optional_input` here to make it clear what's happening.